_modules/sesceph/__init__.py

```diff
@@ -1085,3 +1095,80 @@ def ceph_version():
     u.ceph_version_refresh()
     p = presenter.mdl_presentor(m)
     return p.ceph_version()
+
+
+def cluster_quorum(**kwargs):
+    """
+    Get the cluster status
+
+    CLI Example:
+
+        salt '*' sesceph.cluster_status \\
+                '[rgw.]\n\tkey = AQA/vZ9WyDwsKRAAxQ6wjGJH6WV8fDJeyzxHrg==\n\tcaps rgw = \"allow *\"\n' \\
```

Comment:
Yes definitely, but not a very serious one, as adding the key will not break things.